commands/root.go

```diff
@@ -1,21 +1,21 @@
 package commands
 
 import (
-	"github.com/bndw/pick/config"
+	c "github.com/bndw/pick/config"
 	"github.com/spf13/cobra"
 )
 
 // Global pick config
-var Config *config.Config
+var config *c.Config
 
 var rootCmd = &cobra.Command{
 	Use:   "pick",
 	Short: "pick is a minimal password manager",
 	Long:  "pick is a minimal password manager",
 }
 
-func Execute(cfg *config.Config) error {
-	Config = cfg
+func Execute(cfg *c.Config) error {
+	config = cfg
```

Comment:
I'd rather take the tradeoff of overriding the import package to `c` and having a global named `config`, than renaming the global `safeConfig`.

This could problematic if we need to import the `config` package anywhere else in `commands`, but I do not think that we will.